src/MarginTrading.MarketMaker/Services/ExtPrices/Implementation/GenerateOrderbookService.cs

```diff
@@ -87,73 +86,81 @@ public Orderbook OnNewOrderbook(ExternalOrderbook orderbook)
                 return null;
             }
 
-            var (result, primaryExchange) = OnNewOrderbookInternal(orderbook);
-            LogCycle(orderbook, watch, primaryExchange, result == null);
+            var (result, primaryExchange, problem) = OnNewOrderbookInternal(orderbook);
+            LogCycle(orderbook, result, watch, primaryExchange, problem);
```

Comment:
I think we should have option to disable this logging.

Reply:
Why?